src/main/kotlin/com/swm_standard/phote/controller/ExamController.kt

```diff
@@ -61,6 +63,19 @@ class ExamController(
         return BaseResponse(msg = "문제 풀이 채점 성공", data = response)
     }
 
+    @Operation(summary = "regradeExam", description = "(강사가) 시험 정오답 재채점")
+    @SecurityRequirement(name = "bearer Auth")
+    @PostMapping("/exam/{examId}/{userId}")
+    fun regradeExam(
+        @PathVariable(required = true) examId: UUID,
+        @PathVariable(required = true) userId: UUID,
```

Comment:
```suggestion
        @PathVariable(required = true) memberId: UUID,
```
저희 `userId` -> `memberId` 로 통일 어떠신가요 ~~~~ auth 쪽은 나중에 내가 바꿔놓을게